scripts/deploy.sh

```diff
@@ -87,7 +87,11 @@ echo "$PATH" >> ~/.bashrc
 
 # Setup cloud provider profile
 # TODO: check which cloudprovider
-/bin/bash $SCRIPTS_DIR/aws/setup.sh
+if [ -n "$PROVIDER_FLAG" ]; then
```

Comment:
What do you think of `NO_PROVIDER=true` or `SKIP_PROVIDER` or `PROVIDERS="none"` instead of  `PROVIDER_FLAG`?

Reply:
SKIP_PROVIDER or PROVIDERS="none" looks good.

Reply:
+1 for SKIP_PROVIDER or PROVIDERS="none"

Reply:
ok changed the code to below 

```
if [ "$PROVIDERS" == "none" ]; then
  echo "running bitops for non-default provider..."
else
  /bin/bash $SCRIPTS_DIR/aws/setup.sh
fi
```